### ISSUE TITLE
Walkabout Mini Golf

### ISSUE THREAD

**Issue by newend** [LibreVR/Revive#1738]
**Describe the bug**
When launching the most recent version of the app through revive, the game never loads into vr.

**To Reproduce**
N/A

**Expected behavior**
N/A

**Environment (please complete the following information):**
 - App: 
 -Walkabout Mini Golf
 - OS: [e.g. Win10]
 - Headset: Index

**Versions (please complete the following information):**
 - Revive: most recent
 - SteamVR: most recent

**Additional context**

**Comment by ElBatallitas** [comment]
Same, not working on Pimax.

**Comment by petermg** [comment]
Same here.

**Comment by TomFanella4** [comment]
I was able to get it working using [this approach.](https://www.youtube.com/watch?v=WNKfYgxJLDc). Hope this helps!

**Comment by sh4d0vv1** [comment]
@TomFanella4 ya we had issues with new updates on population one and dev helped us fix the issue. But said that 2
3.0 fixed the issue but it never has for me .. I and many others I helped did the same issue have to redo the steps every week because population one updates weekly and reverts to new runtime which doesn't work so always have to paste the dll file in weekly and poof works

**Comment by TomFanella4** [comment]
@sh4d0vv1 Yeah the devs need to cut a new release of 3.0 that includes this change https://github.com/LibreVR/Revive/commit/9dcf65077da891f61c55506937353a27159e6ea7. I mentioned that on their discord server but no one has done it yet. Not sure who to contact to make that happen.

**Comment by TomFanella4** [comment]
I created a new issue here. It would be super helpful if you could vote it up! https://github.com/LibreVR/Revive/issues/1752
